design/appendix/attack-scenarios.tex

```diff
@@ -0,0 +1,163 @@
+\chapter{Attack scenarios}\label{chap:attack scenarios}
+
+The goal of this \namecref{chap:attack scenarios} is to give an overview how various aspects of the Peras rules/requirements are necessicated by concrete attacks that could occur otherwise.
+\todo{Figure out if/in what capacity this should end up in the final version}
+
+\section{Simple attacks}
+
+\subsection{Abstaining during voting}\label{sec:abstaining during voting}
+
+An adversary with $\alpha$ stake will have $\alpha\cdot\perasN$ votes on average, so if they simply abstain, the honest nodes will likely fail to reach quorum even when they all vote for the same block when \[ \alpha \ge 1-\frac\perasQuorum\perasN = 0.25 \; \]
+Therefore, Peras doesn't provide fast settlement in the presence of an adversary with $\alpha\ge 25\%$.
+
+As usual, this is assuming full honest participation; smaller participation levels can only tolerate proportionally weaker adversaries.
+
+\subsection{Honest vote splitting}\label{sec:honest vote splitting}
+
+\todo{Mention Russell's suggested modification here}
+In the pre-alpha version of Peras, honest nodes vote for the newest block on their selection that is at least \perasBlockMinSlots{} slots old.
+It is relatively easy for an adversary to cause different honest nodes to vote for different blocks.
+
+For example, this allows an adversary with $\alpha<0.25$ to prevent quorum formation with decent probability (which they couldn't do by simply abstaining) by splitting the honest votes such that neither block gets \perasQuorum{} votes, causing the system to enter a cooldown period.
+Future versions of Peras will improve on this front by adding a dedicated \emph{pre-agreement} mechanism that \emph{avoids} a cooldown in this scenario.
+
+Write $\operatorname{tip}(C)$ for the tip of a chain $C$, and $\operatorname{trunc}(s,C)$ for the prefix of $C$ up until (but excluding) slot $s$.
+
+Concretely, consider the voting phase at the beginning of round $r$ starting in slot $s$, with all honest nodes having selected chain $C$ just before.
+
+Suppose that the adversary with stake $\alpha$ has a chain $D$ that is preferrable to $C$, and $V_C \neq V_D$ where \[V_c = \operatorname{tip}(\operatorname{trunc}(s-\perasBlockMinSlots,c))\] for $c\in\{C,D\}$.
```

Comment:
The difference between `V_C` and `V_c` is quite small, I'd rather use an entirely different letter than `c` for `... \in {C, D}`.